test/libsolidity/smtCheckerTests/array_members/push_function_return_value.sol

```diff
@@ -0,0 +1,20 @@
+contract C {
+	bytes x;
+
+	function getX() internal view returns (bytes storage) {
+		return x;
+	}
```

Comment:
How about functions taking `storage` arguments? I don't see any cases covering that in `smtCheckerTests/array_members/` (I haven't inspected all of them but none of those whose names refer to functions or arguments seem to have that).

Reply:
Yea there are a bunch of tests with storage/memory arguments in `types`, for example `types/array_aliasing_storage_1.sol`